beacon_node/eth2-libp2p/src/behaviour.rs

```diff
@@ -140,16 +140,44 @@ impl<TSubstream: AsyncRead + AsyncWrite, E: EthSpec> Behaviour<TSubstream, E> {
     }
 }
 
-impl<TSubstream: AsyncRead + AsyncWrite, E: EthSpec> NetworkBehaviourEventProcess<Discv5Event>
-    for Behaviour<TSubstream, E>
+impl<TSubstream: AsyncRead + AsyncWrite> NetworkBehaviourEventProcess<IdentifyEvent>
+    for Behaviour<TSubstream>
+{
+    fn inject_event(&mut self, event: IdentifyEvent) {
+        match event {
+            IdentifyEvent::Identified {
+                peer_id, mut info, ..
+            } => {
+                if info.listen_addrs.len() > 20 {
```

Comment:
Yeah this is kind of temporary. I removed Identify, but because of others using the go-daemon it useful for debugging. This is only used for debugging and doesn't really touch any functionality. 
It will be removed in later revisions I imagine.